packages/admin/cms-admin/src/userPermissions/user/UserPageToolbar.tsx

```diff
@@ -38,7 +44,28 @@ export const UserPermissionsUserPageToolbar = ({ userId }: { userId: string }) =
             </ToolbarTitleItem>
             <ToolbarFillSpace />
             <ToolbarActions>
-                <StartImpersonationButton userId={userId} />
+                <CrudMoreActionsMenu
+                    slotProps={{
+                        group: { groupTitle: null },
+                    }}
```

Comment:
A task already exists to hide the group-title if there is only one group: [COM-1174](https://vivid-planet.atlassian.net/browse/COM-1174)
So this and the change in `CrudMoreActionsMenu` should not be needed. 

```suggestion
```